### PR TITLE
Update productivity-score.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note of users being able to view some of their productivity insights

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3563

File: microsoft-365/admin/productivity/productivity-score.md [hunk 1]
```diff
@@ -133,6 +133,10 @@ This report helps business leaders understand:
 
 [Learn more about Microsoft Graph](https://docs.microsoft.com/graph/)
 
+> [!NOTE]
+> User's also have option to vieww productivity insights from [Workplace Analytics](https://docs.microsoft.com/workplace-analytics/myanalytics/use/dashboard-2)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/productivity/productivity-score.md]:
```suggestion
> Users also have the option to get productivity insights from the [MyAnalytics dashboard](https://docs.microsoft.com/workplace-analytics/myanalytics/use/dashboard-2).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**